WebRoot/skins/_base/base4/skin.css

```diff
@@ -1356,3 +1356,17 @@ div[name='_mergeBtnContainerId'] .ZCalToggleBtnSelected .ZWidgetTitle {
 .ZmSharePropsDialog .ZmShareDialogSection .ZmShareReply > div {
     margin-top:0 !important; /* has to use !important to override inline styles */
 }
+
+/* Rotate animation keyframe */
+@keyframes spin {
+    0% {
+      @transform@: rotate(0deg);
+    }
+    100% {
+      @transform@: rotate(360deg);
+    }
+}
+
+.ZmSpinElement {
```

Comment:
can we move this to zm.css?

Reply:
Sure, we can. Let me move it.

Reply:
Moved: https://github.com/Zimbra/zm-web-client/pull/210/commits/78c4af87182fc5668dc6ffd0d206a382e8044f4b

Thanks

Reply:
Thanks